rustfmt.toml

```diff
@@ -0,0 +1,5 @@
+comment_width = 100
+edition = "2021"
+group_imports = "One"
```

Comment:
Is `One` even an [option](https://rust-lang.github.io/rustfmt/?version=v1.4.38&search=group#group_imports)?

Reply:
[It is](https://rust-lang.github.io/rustfmt/?version=master&search=group#group_imports) at least with nightly. Took this example from the spl library

Reply:
I see. Let it be.